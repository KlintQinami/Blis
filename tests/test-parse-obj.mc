int main()
{
  vec3[] verts;
  int[] faces;
  if (!read_obj("./kitten.obj", verts, faces)) {
    print("error reading obj\n");
    return 1;
  }

  printi(length(verts));
  printi(length(faces) / 3);

  return 0;
}
