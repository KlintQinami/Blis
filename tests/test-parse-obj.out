14472
28944
